### PR TITLE
PR for review test

### PR DESCRIPTION
**Pull request by jg020135**
check code

### CODE REVIEW

File: another.c [hunk 1]
```diff
@@ -2,4 +2,6 @@ another file from tutorial
 
 pull request test from jg020135
 
-added from both repositories.
\ No newline at end of file
+added from both repositories.
+  
+<code>abracadabra</code>
```
Comment (0nmyhead) [inline another.c]: spell in code needs to be changed.

**Review by 0nmyhead — CHANGES_REQUESTED** [review]
check comment in code pls.

File: another.c (hunk 1 above)
Comment (jg020135) [reply another.c]: i fixed spell.

Comment (0nmyhead) [reply another.c]: seems good to merge

**Review by 0nmyhead — APPROVED** [review]
spell changed.

**Pull request merged**